google-cloud-service_directory/README.md

```diff
@@ -1,40 +1,42 @@
-# Ruby Client for Service Directory API
+# Ruby Client for the Service Directory API
+
+API Client library for the Service Directory API
 
-[Service Directory API][Product Documentation]:
 Service directory is the single place to register, browse, and resolve application services.
-- [Client Library Documentation][]
-- [Product Documentation][]
 
-## Quick Start
-In order to use this library, you first need to go through the following
-steps:
+Actual client classes for the various versions of this API are defined in
+_versioned_ client gems, with names of the form `google-cloud-service_directory-v*`.
+The gem `google-cloud-service_directory` is a convenience wrapper library that brings the
+verisoned gems in as dependencies, and provides high-level methods for
+constructing clients.
 
-1. [Select or create a Cloud Platform project.](https://console.cloud.google.com/project)
-2. [Enable billing for your project.](https://cloud.google.com/billing/docs/how-to/modify-project#enable_billing_for_a_project)
-3. [Enable the Service Directory API.](https://console.cloud.google.com/apis/library/servicedirectory.googleapis.com)
-4. [Setup Authentication.](https://googleapis.dev/ruby/google-cloud-service_directory/latest/file.AUTHENTICATION.html)
+View the [Client Library Documentation](https://googleapis.dev/ruby/google-cloud-service_directory/latest)
+for this library, google-cloud-service_directory, to see the convenience methods for
+constructing client objects. Reference documentation for the client objects
+themselves can be found in the client library documentation for the versioned
+client gems:
+[google-cloud-service_directory-v1beta1](https://googleapis.dev/ruby/google-cloud-service_directory-v1beta1/latest).
+
+See also the [Product Documentation](https://cloud.google.com/service-directory)
```

Comment:
This URL currently returns 404.

Reply:
Known issue. It will be available shortly.